packages/sdk-drop-in/tests/iframe.test.ts

```diff
@@ -0,0 +1,45 @@
+import { test } from '@playwright/test';
+
+const getMockConfig = () => ({
+  client_id: '2afafd88-7b8c-4ed2-b0f0-652c0db600b7',
+  client_secret: 'ae6b5ece-51a1-4fcf-97d2-d10bb0de0612',
+  entity_user_id: '0771f748-2e98-400e-adb4-20ebefd77f5f',
+  grant_type: 'entity_user',
+});
```

Comment:
We should not expose secrets, even development. Usually, we use MSW (as in `@monite/sdk-react` package). But this package is not yet ready to implement MSW tests probably. So, for the local environment let's use `.env.local` file with predefined env variables. For the CI stage, we will use GitHub secrets.